neqo-http3-server/src/main.rs

```diff
@@ -104,9 +110,37 @@ fn process_events(server: &mut Http3Server) {
 
                 let response = headers.and_then(|h| {
                     h.iter().find(|&(k, _)| k == ":path").and_then(|(_, path)| {
-                        match path.trim_matches(|p| p == '/').parse::<usize>() {
-                            Ok(v) => Some(vec![b'a'; v]),
-                            Err(_) => Some(default_ret),
+                        if args.qns_mode {
+                            let mut file_path = PathBuf::from("/www");
+                            file_path.push(path.trim_matches(|p| p == '/'));
+
+                            OpenOptions::new()
+                                .read(true)
+                                .open(&file_path)
+                                .map_err(|_e| eprintln!("Could not open {}", file_path.display()))
+                                .ok()
+                                .and_then(|mut f| {
+                                    let mut data = Vec::new();
+                                    match f.read_to_end(&mut data) {
```

Comment:
Could reading all data into memory be problem if we decide to test 1Gbps? I guess we would need to serve at least 500MB files to let the test run for at least couple of seconds.

Reply:
500MB? Not that big, should work ok :slightly_smiling_face: but yes that's how http3's current server API works, response is given entire data to send in one chunk.